code/modules/mob/living/simple_animal/friendly/mouse.dm

```diff
@@ -33,9 +39,9 @@
 
 /mob/living/simple_animal/mouse/Life()
 	..()
-	if(!stat && prob(speak_chance))

```

Comment:
Sleeping no, but dead mice possibly. That would have technically allowed a mouse to still squeek if its controlled by a player AND dead, AND the player hasn't left the corpse as a ghost yet. A kinda minor edge case, but i'm fixing it anyway